### ISSUE TITLE
AttributeError: 'WebDriver' object has no attribute 'get_device_time'

### ISSUE THREAD

**Issue by zhengyu88** [appium/python-client#483]
## The problem
When I tried to use `driver.get_device_time()` method, it threw this error:

    AttributeError: 'WebDriver' object has no attribute 'get_device_time'

## Environment

* Appium version (or git revision) that exhibits the issue: 1.16.0
* Last Appium version that did not exhibit the issue (if applicable): N/A
* Desktop OS/version used to run Appium: Windows 10
* Node.js version (unless using Appium.app|exe): v8.11.3
* Npm or Yarn package manager: npm
* Mobile platform/version under test: android 9
* Real device or emulator/simulator: real device
* Appium CLI or Appium.app|exe: Appium CLI

## Details

1. I start appium service by using this command(before that I updated appium to 1.16.0 by using `npm -g update appium`):

        appium -p 4723

2.  I set up the driver on my python script:

        driver = webdriver.Remote('http://localhost:4723/wd/hub', desired_caps)

3.  I am trying to get device time by using:

         driver.get_device_time("YYYY-MM-DD HH")

4.  I ran the python script and it threw the following error:

        AttributeError: 'WebDriver' object has no attribute 'get_device_time'

However, if I use the appium desktop app(`1.16.0`) instead of starting appium on cmd line, it worked fine.

**Comment by KazuCocoa** [comment]
What version of python client did you use?

**Comment by zhengyu88** [comment]
python: 2.7.15 @KazuCocoa

**Comment by ki4070ma** [comment]
At least on my macOS, it works.

```
➜  appium-sample git:(bdd) ✗ appium --version               
1.16.0
➜  appium-sample git:(bdd) ✗ python --version               
Python 2.7.16
➜  appium-sample git:(bdd) ✗ pip show Appium-Python-Client  
Name: Appium-Python-Client
Version: 0.49
Summary: Python client for Appium
Home-page: http://appium.io/
Author: Isaac Murchie
Author-email: isaac@saucelabs.com
License: Apache 2.0
Location: /Users/atsushimori/.pyenv/versions/2.7.16/lib/python2.7/site-packages
Requires: selenium
Required-by: 
➜  appium-sample git:(bdd) ✗ python SampleAndroidTest_py2.py
***start
2020-01-18T13:59:08+09:00
2020-01-18
2020-01-18 13
***end
```

```
#!/usr/bin/python
# -*- coding: utf-8 -*-

from appium import webdriver

caps = {
    'platformName': "Android",
    'deviceName': "Android Emulator",
    'appPackage': "com.android.settings",
    'appActivity': ".SubSettings",
    'automationName': "uiautomator2",
}

if __name__ == '__main__':
    driver = webdriver.Remote('http://localhost:4723/wd/hub', caps)

    print('***start')
    print(driver.get_device_time())
    print(driver.get_device_time("YYYY-MM-DD"))
    print(driver.get_device_time("YYYY-MM-DD HH"))
    print('***end')

    driver.quit()
```

@zhengyu88 
Could you provide appium server log?

**Comment by ki4070ma** [comment]
@zhengyu88 

Sorry, from below info, it doesn't seem due to server side.
You don't need to take server log for now.

* AttributeError: 'WebDriver' object has no attribute 'get_device_time'
* However, if I use the appium desktop app(1.16.0) instead of starting appium on cmd line, it worked fine.

Could you provide below info?

1. `$appium --version`
2. `$python -m pip show Appium-Python-Client`

**Comment by zhengyu88** [comment]
`appium --version`: 1.16.0

`python -m pip show Appium-Python-Client`:

    Name: Appium-Python-Client
    Version: 0.28
    Summary: Python client for Appium 1.5
    Home-page: http://appium.io/
    Author: Isaac Murchie
    Author-email: isaac@saucelabs.com
    License: Apache 2.0
    Location: e:\python27\lib\site-packages
    Requires: selenium
    Required-by:

**Comment by ki4070ma** [comment]
@zhengyu88 
Thanks for your info.

`v0.28` doesn't have `get_device_time()`, it's available from `v0.35`.
Please update `Appium-Python-Client` and try again.

[Notes] Added format to device_time as argument https://github.com/appium/python-client/pull/312

Not sure why it works with below environments from current info.

> However, if I use the appium desktop app(1.16.0) instead of starting appium on cmd line, it worked fine.

**Comment by zhengyu88** [comment]
I upgraded` Appium-Python-Client` to 0.49, and it was working, thank you so much!
